cannot `agoric set-defaults` in docker for instagoric

## Describe the bug

While trying to get a performance baseline for state sync experiments (#5934), rebooting an instagoric network failed to run `agoric set-defaults ...`.

## To Reproduce
Steps to reproduce the behavior:
1. `docker pull agoric/agoric-sdk:dev`
2. `docker run --rm -ti --entrypoint /bin/bash agoric/agoric-sdk:dev`
3. `agoric`
4. See error

```
$ docker run --rm -ti --entrypoint /bin/bash agoric/agoric-sdk:dev 
root@f9d2cf9e3495:~# agoric
node:internal/errors:478
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/usr/src/agoric-sdk/node_modules/@agoric/cosmic-proto/dist/agoric/swingset/msgs.js' imported from /usr/src/agoric-sdk/packages/agoric-cli/src/publish.js
    at new NodeError (node:internal/errors:387:5)
    at finalizeResolution (node:internal/modules/esm/resolve:330:11)
    at moduleResolve (node:internal/modules/esm/resolve:907:10)
    at defaultResolve (node:internal/modules/esm/resolve:1115:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:837:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:40)
    at link (node:internal/modules/esm/module_job:75:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

## Expected behavior

`agoric set-defaults ag-chain-cosmos /state/agoricloadtest-14/config` should work.

## Platform Environment
 - what OS are you using? what version of Node.js?
 - is there anything special/unusual about your platform?
 - what version of the Agoric-SDK are you using? (run `git describe --tags --always`)

4d802f2

```
$ docker pull agoric/agoric-sdk:dev
dev: Pulling from agoric/agoric-sdk
Digest: sha256:7cbd34dfaa0c7929a5470572c5d9589000c78d36e6c7b9a612929e43ff28de18
Status: Image is up to date for agoric/agoric-sdk:dev
docker.io/agoric/agoric-sdk:dev

$ docker run --rm -ti agoric/agoric-sdk:dev version --long | grep commit
commit: 4d802f2
```

## Additional context

Something about recent changes to cosmic-proto doesn't seem to work right in this environment.

cc @michaelfig @turadg @arirubinstein @kriskowal

--- a/.dockerignore
+++ b/.dockerignore
@@ -10,7 +10,6 @@ packages/stat-logger
 **/deployment.json
 **/vars.tf
 **/*.log
-**/dist
 **/build
 **/__pycache__
 **/*.egg-info

